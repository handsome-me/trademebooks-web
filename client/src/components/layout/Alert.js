import React from 'react'
import ReduxToastr from 'react-redux-toastr'

const Alert = () => {
  return (
    <ReduxToastr
      timeOut={3000}
      newestOnTop={false}
      preventDuplicates
      position="top-right"
      getState={(state) => state.toastr}
      transitionIn="fadeIn"
      transitionOut="fadeOut"
      closeOnToastrClick
    />
  )
}

export default Alert
