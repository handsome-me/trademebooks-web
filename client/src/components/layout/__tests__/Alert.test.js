import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import { Provider } from 'react-redux'

import store from '../../../store'
import Alert from '../Alert'

describe('Component: Alert', () => {
  it('should match the snapshot', () => {
    const snapshot = renderer.create(
      <Provider store={store}>
        <MemoryRouter>
          <Alert />
        </MemoryRouter>
      </Provider>
    ).toJSON()

    expect(snapshot).toMatchSnapshot()
  })
})
