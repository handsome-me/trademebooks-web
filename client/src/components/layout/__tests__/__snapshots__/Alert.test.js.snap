// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Alert should match the snapshot 1`] = `
<div
  aria-live="assertive"
  className="redux-toastr"
>
  <div>
    <div
      className="top-left"
      style={Object {}}
    />
    <div
      className="top-right"
      style={Object {}}
    />
    <div
      className="top-center"
      style={Object {}}
    />
    <div
      className="bottom-left"
      style={Object {}}
    />
    <div
      className="bottom-right"
      style={Object {}}
    />
    <div
      className="bottom-center"
      style={Object {}}
    />
  </div>
</div>
`;
